docs/usage/delete.rst

```diff
@@ -7,6 +7,12 @@ Examples
     # delete a single backup archive:
     $ borg delete /path/to/repo::Monday
 
+    # delete all archives whose names begin with the machine's hostname
+    $ borg delete --prefix $HOSTNAME /path/to/repo
```

Comment:
borg prune example uses this `--prefix='{hostname}-'`  (please try if it works for borg delete, if not, file a bug)

the `-` after the hostname is important, otherwise e.g. a host `server` would also delete the `server2-*` archives (not just the `server-*` archives), which is likely not wanted.

Reply:
`--prefix='{hostname}-'` works.